timemachine/ff/handlers/deserialize.py

```diff
@@ -21,8 +21,8 @@ def deserialize_handlers(obj):
 
     handlers = []
 
-    protein_ff = obj_dict.pop(constants.PROTEIN_FF_TAG, constants.DEFAULT_PROTEIN_FF)
-    water_ff = obj_dict.pop(constants.WATER_FF_TAG, constants.DEFAULT_WATER_FF)
+    protein_ff = obj_dict.pop(serialization_format.PROTEIN_FF_TAG, constants.DEFAULT_PROTEIN_FF)
+    water_ff = obj_dict.pop(serialization_format.WATER_FF_TAG, constants.DEFAULT_WATER_FF)
```

Comment:
Small surprise / possible code-smell: `deserialize_handlers()` hard-codes dependence on DEFAULT_PROTEIN_FF and DEFAULT_WATER_FF, but `serialize_handlers(..., protein_ff, water_ff)` exposes these as parameters.

Reply:
Could add the default values when there are no protein/water specified in the handlers in the signature? Otherwise this seems reasonable to me, if there is no water/protein ff specified fall back to the defaults. We could also put explicit water/protein ffs in our default FFs

Reply:
Added in https://github.com/proteneer/timemachine/pull/989/commits/171c17ce29f30f26a6e839046fe49c68eaa13055

Reply:
Reverted in https://github.com/proteneer/timemachine/pull/989/commits/be35ee1d02559ed6680f1246d308ef05b2268d39 to avoid messing things up, following https://github.com/proteneer/timemachine/pull/989#discussion_r1158931631